src/Selenium2Library/keywords/cookie.py

```diff
@@ -1,46 +1,58 @@
-from .keywordgroup import KeywordGroup
+from Selenium2Library.robotlibcore import keyword
 
 
-class CookieKeywords(KeywordGroup):
+class CookieKeywords(object):
```

Comment:
Just for my own understanding:
Why does this class inherit from object, but many other keyword classes inherit from Base?
I see that it isn't necessary since there is no logging for these keywords.  I just want to understand the principal behind the choice.
Is it simplicity over uniformity?